docs/crystal_for_rubyists/metaprogramming_help.md

```diff
@@ -115,7 +120,7 @@ This example makes use of the `macro extended` hook. This hook is called wheneve
 
 ### Generating Methods via `method_missing` Macro
 
-Sample code + explanation
+[Here](https://github.com/zw963/hashr/blob/master/src/hashr.cr) is a very simple example for how to use method_missing macro, you can check the spec code [here](https://github.com/zw963/hashr/blob/master/spec/hashr_spec.cr), it shows what method_missing macro does.
```

Comment:
I'd rather embed the examples here directly. That makes it easier to skim and the reader doesn't have to navigate from the page.

Reply:
Done, copy the code here. probably need more explain on those code, otherwise, user may still can't understand what is this doing.